### DISCUSSION TITLE
ClubLog synchronization

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#1578] [Q&A] [answered]
Hi guys,

I'm trying to setup ClubLog syncrhonization. 
In Wiki, there are two types of synchronization. Let's say general and realtime.
I created cron jobs for both of them.

In Cloudlog I have two locations, "home" with about 720 QSOs, and "roving", with only 3 QSOs (4O4A was the call for roving).

After the cron jobs, Clublog shows only those 3 QSOs from roving, and none of QSOs since my last manual upload to ClubLog (about two months ago).

What I am missing?

Regards,
Dragan, 4O4A

**Comment by magicbug** [comment]
If your using station_logbooks branch clublog uploadings broken at the moment

**Reply by 4O4A** [reply]
Frankly, I'm not sure what I am using. I have no problem to start all over again with installation and settings. 
Actually, I have only one problem: please remind me, when installation process starts, what is "directory"? :) 
Absolute path to server doesn't work, relative to URL in field below, partial this, partial that... all I get is "directory xxxx cannot be found" :)

**Reply by 4O4A** [reply]
Please disreagard question about "directory". 
Having subdomain is probably not good idea during istallation. :)

**Comment by 4O4A** [comment]
I am using "master" fork. Now, brand new installation. Only one station location for the time being.
Setup cron jobs for LoTW, Clublog. LoTW works fine. ClubLog still shows nothing new. 
This is cron job code, once per minute: 
`curl --silent https://leo.4o4a.com/clublog/upload/xxxxxxxxxx &>/dev/null` 
where xxxxxxxx is my cloudlog username.

**Comment by 4O4A** [comment]
This is the error I get if I manually go to URL from cron job:

`operation aborted by callbackError`

**Comment by magicbug** [comment, marked as answer]
Hopefully this is now resolved in [Stations_logbook] branch.

**Reply by 4O4A** [reply]
thank you, I will test it

**Reply by 4O4A** [reply]
When I go to this URL: https://leo.4o4a.com/clublog/upload/4o4a
I got this response in browser:

Failed to connect to clublog.org port 443: Connection refusedError

**Reply by phl0** [reply]
That would be an issue with the Clublog servers I guess. Not something that Cloudlog could handle.

**Reply by 4O4A** [reply]
I have two branches on my server: Master and Station_Logbooks.

Master: 
https://leo.4o4a.com

Stations_Logbooks:
https://4o4a.com/leo20/index.php

Mater: using .htaccess
Station_Logbooks: doesn't use .htaccess

Master: does not upload to ClubLog
Station_Logbooks: uploads to ClubLog

Master: uploads to LoTW
Station_Logbooks: doesn't upload to LoTW

Cron job for LoTW for Master:
* | * | * | * | * | curl --silent https://leo.4o4a.com/lotw/lotw_upload &>/dev/null

Cron job for LoTW for Stations_Logbooks
* | * | * | * | * | curl --silent https://4o4a.com/leo20/index.php/lotw/lotw_upload &>/dev/null

What am I missing?
